setup.py

```diff
@@ -107,6 +107,7 @@
     "jax>=0.2.8",
     "jaxlib>=0.1.65",
     "jieba",
+    "keras<2.7.0", # Remove when they fix their release
```

Comment:
As said offline, proposal
```suggestion
    "keras!=2.7.0", # Remove when they fix their release
```